test.sh

```diff
@@ -220,40 +226,29 @@ if [ "$TEST_CASE" == 0 ] || [ "$TEST_CASE" == 1 ]; then
   echo "----------------------------------------------------------"
   echo
   echo "*** 1-a: Pod to Pod (Same Node) ***"
-  if [ "$DEBUG_TEST" == true ]; then
-    echo "DEBUG - BEGIN"
-    echo
-
-    echo "kubectl exec -it $LOCAL_CLIENT_POD -- ping $SERVER_IP -c 3"
-    kubectl exec -it $LOCAL_CLIENT_POD -- ping $SERVER_IP -c 3
-    echo
-
-    echo "DEBUG - END"
-    echo
-  fi
+  echo
 
   echo "kubectl exec -it $LOCAL_CLIENT_POD -- $CURL_CMD \"http://$SERVER_IP:$SERVER_POD_PORT/\""
   TMP_OUTPUT=`kubectl exec -it $LOCAL_CLIENT_POD -- $CURL_CMD "http://$SERVER_IP:$SERVER_POD_PORT/"`
   process-curl-output "${TMP_OUTPUT}" "${POD_SERVER_STRING}"
-  echo
+  
+  echo "*** 1-a: Pod to Pod (Same Node) TRACE ***"
+  ./ovnkube-trace -loglevel=5 -tcp -ovn-config-namespace=$OVN_K_NAMESPACE $SSL_ENABLE -dst=$SERVER_POD_NAME -dst-port=$SERVER_POD_PORT -src=$LOCAL_CLIENT_POD -kubeconfig=$KUBECONFIG 2> ovn-traces/pod2pod-same-node.txt
```

Comment:
On clean install, directory doesn't exist. I get the following and no output files generated:
```
*** 1-a: Pod to Pod (Same Node) TRACE ***
./test.sh: line 236: ovn-traces/pod2pod-same-node.txt: No such file or directory
*** END OF TRACE (see ovn-traces/pod2pod-same-node.txt for full detail
```

Reply:
Ah yeah I must have done this wrong in the git-ignore I want to ignore all files in the directory but not the directory.

Reply:
I don't think anything ever created that directory. You either need to checkin an empty directory, or add `mkdir -p ovn-traces` somewhere (like launch.sh or test.sh).

Reply:
You merge didn't fix it.

Reply:
Should be good now